src/pages/Account.vue

```diff
@@ -5,7 +5,7 @@
             <div class="content fullwidth">
                 <h1>Requested Payments</h1>
 
-                <transition name="fade"><div v-if="errorMessage" v-bind:class="[goodFeedback ? 'goodmessage' : 'message' ]">{{ errorMessage }}</div></transition>
+                <transition name="fade"><div v-if="errorMessage" v-bind:class="[goodFeedback ? 'good.message' : 'message' ]">{{ errorMessage }}</div></transition>
```

Comment:
Is good.message the right syntax to add class="good message"?
Would it be better to default to message and add the good optionally?

Also confused by the semantics - this is an errorMessage - so not good. If it is a good message maybe it should not be passed in a var called errorMessage?